t/base.t

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/perl -w
 
 use strict;
+#use utf8;
```

Comment:
I presume you would just delete this line instead of commenting out; likewise in another spot

Reply:
Yes, mainly left it until you told me it wasn't dumb. :-)

Reply:
Removed in 608dbb7.